great-idea-website/css/index.css

```diff
@@ -63,4 +63,116 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+
+.container {
+	width: 876px;
+	margin: auto;
+}
+
+header nav {
+	display: inline-block;
+	margin-top: 35px;
+}
+
+
+.cover .cover-content {
+	margin-bottom: -9px;
+	display: inline-block;
+}
+
+
+header nav a {
+	margin-right: 35px;
+    margin-left: 27px;
+	font-size: 14px;
+	color: gray;
+	text-decoration: none;
+}
+
+img .logo {
+	margin-left: 110px;
+}
+
+.cover .cover-content {
+	vertical-align: top;
+	font-size: 40px;
+	margin-top: 120px;
+	text-align: center;
+	padding: 0px 50px;
+}
+
+.cover img {
+	display: inline-block;
+	margin-top: 69px;
+}
+
+.title {
+	margin-bottom: -6px;
+}
+
+button {
+	background: white;
+	/* vertical-align: top; */
+	margin-top: 1px;
+	padding: 3px 24px;
+	font-size: 12px;
+	border: 1px solid black;
+	border-radius: 5px;
+	cursor: pointer;
+	background: white;
+}
+.main-content {
+	border-top: 1px solid black;
+	margin-top: 48px;
+	padding-top: 33px;
+	margin-bottom: -50px;
+  }
+
+
+.main-content .top-content .text-content {
+	width: 44%; 
```

Comment:
This is good. The width depends on how wide your middle margin is and its based on what you want.